src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_postgres.py

```diff
@@ -1593,7 +1593,7 @@ def _update_login(server_name, resource_group_name, auth_config, password_auth,
 
 
 # pylint: disable=too-many-instance-attributes, too-few-public-methods, useless-object-inheritance
```

Comment:
[`useless-object-inheritance`](https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/useless-object-inheritance.html) can be removed.

```suggestion
# pylint: disable=too-many-instance-attributes, too-few-public-methods
```

Reply:
Fixed